package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.ts",
   "scripts": {
     "start": "tsc && node dist/index",
-    "dev": "tsc && nodemon src/index.ts",
+    "dev": "ts-node-dev --transpile-only --no-notify --exit-child src/index.ts",
```

Comment:
Is this supposed to solve the issue you mentioned in the background section? Personally, I don't get that error when starting the app.

Reply:
@ceelogre 
The above issue is a result of a user not installing the `ts-node` as a global dependency which rarely people do when not mentioned in the instructions. Since they have `Typescript` when the error rise it is hard to pinpoint since the module is already installed.

This command also removes the need for `nodemon` to watching apart from such cases since the command will perform a reload upon file change. These modules need to be added in the dependency list as of now `nodemon` and `ts-node` are not in the list but assume the user have them installed globally.

Reply:
`nodemon` already has built-in support for `ts`. Checkout [this](https://github.com/remy/nodemon/releases/tag/v1.19.0) release message.

Reply:
@ceelogre 
Nodemon does support the ts and serve the same purpose as the `ts-node-dev`, I will adjust the instructions accordingly.

Though the need for the `ts-node-dev` is that it perform live reload when files changes. This makes it an all in one typescript solution that compile and watch the  files. That is the sole reason behind.